package.json

```diff
@@ -466,18 +466,20 @@
         "compile": "webpack --mode development && npm run buildScripts",
         "recompile": "npm run clean && npm run compile",
         "watch": "npm run buildScripts && tsc -watch -p ./",
-        "postinstall": "ts-node ./build-scripts/generateServiceClient.ts",
+        "postinstall": "ts-node ./build-scripts/generateServiceClient.ts && npm run generateTelemetry",
         "testCompile": "tsc -p ./ && npm run buildScripts",
         "test": "npm run testCompile && ts-node ./test-scripts/test.ts",
         "integrationTest": "npm run testCompile && ts-node ./test-scripts/integrationTest.ts",
         "lint": "tslint --project .",
         "lintfix": "tslint --project . --fix",
         "package": "vsce package",
         "install-plugin": "vsce package -o aws-toolkit-vscode-test.vsix && code --install-extension aws-toolkit-vscode-test.vsix",
+        "generateTelemetry": "node node_modules/@aws-toolkits/telemetry/lib/generateTelemetry.js --input=src/shared/telemetry/vscodeTelemetry.json --output=src/shared/telemetry/telemetry.ts",
         "newChange": "ts-node ./build-scripts/newChange.ts",
         "createRelease": "ts-node ./build-scripts/createRelease.ts"
     },
     "devDependencies": {
+        "@aws-toolkits/telemetry": "0.0.4",
```

Comment:
Not sure what our versioning strategy, but we may want some sort of a caret here to automatically grab updates. Potentially something like `^0.0`, which should be good through `0.1.0`.

Reply:
I made this an exact version on purpose because until this gets going for VSCode and JetBrains normally we are liable to make breaking changes. It also only needs to be updated when we update telemetry, there is no benefit otherwise.